mantidproject/mantid
# Allow log levels to match the lowest of all filter channels

The log level of all the loggers is now set to follow the lowest log level of all of the FilterChannels (File, Console and message display in mantidplot).  It used to track them going down to debug, but never back up again, now it does.

**To test:**
This is tested pretty well in the unit tests in ConfigServiceTest.  However you should also start up Mantidplot, change the log level of the log window and check the correct level filter is applied when running algorithms.

Fixes #14524 
##### RELEASE NOTES

Does not need to be in the release notes.

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.

This should allow the logger.is functionality to work correctly.
